### PR TITLE
fix(@desktop/wallet): add/re-add buy crypto feature

### LINKED ISSUE #4925: add/re-add buy crypto feature
### Description

previously added here https://github.com/status-im/status-desktop/pull/3359

* [ ] Add Buy/Sell button next to Send and Receive where they are right now
* [ ] Add buy crypto functionality as previously implemented in that PR and here https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=797%3A85627

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #4925

### What does the PR do

Re-added the buy/sell crypto feature from https://github.com/status-im/status-desktop/pull/3359

### Affected areas

Wallet

### Screenshot of functionality

https://user-images.githubusercontent.com/60327365/156192046-7da95fdc-0fab-4505-93ff-7d47a939742e.mov

### CODE REVIEW

File: src/app/modules/main/wallet_section/buy_sell_crypto/module.nim [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  self.controller.init()
+  self.view.load()
+
+method isLoaded*(self: Module): bool =
+  return self.moduleLoaded
+
+method viewDidLoad*(self: Module) =
+  self.fetchCryptoServices()
+  self.moduleLoaded = true
+  self.delegate.accountsModuleDidLoad()
```
Comment (alaibe) [inline src/app/modules/main/wallet_section/buy_sell_crypto/module.nim:61]: wrong name here I'd assume ;)

File: src/app/modules/main/wallet_section/buy_sell_crypto/module.nim [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      w.description,
+      w.fees,
+      w.logoUrl,
+      w.siteUrl,
+      w.hostname
+    ))
+  self.view.setItems(items)
+
+method load*(self: Module) =
+  singletonInstance.engine.setRootContextProperty("buySellCrypto", newQVariant(self.view))
```
Comment (alaibe) [inline src/app/modules/main/wallet_section/buy_sell_crypto/module.nim:51]: Could you prefix with wallet section to avoid conflict in name?

**Review by alaibe — APPROVED** [review]
a couple of comment but LGTM

**Review by jrainville — COMMENTED** [review]
Good job. Just fix the small things Anthony found and then it's gucci

**Comment by Khushboo-dev-cpp** [comment]
All done @alaibe and @jrainville . Also used StatusLIstItem instead cause it was so easy to.

**Review by jrainville — APPROVED** [review]
Good job

**Pull request merged**